documentation/src/main/asciidoc/topics/con_performance_partition_handling.adoc

```diff
@@ -0,0 +1,43 @@
+[id='performance-partition-handling_{context}']
+= Network partitions and degraded clusters
+
+{brandname} clusters can encounter split brain scenarios where subsets of nodes in the cluster become isolated from each other.
+Nodes in a network partition can communicate only with other nodes in that partition.
+When this happens, {brandname} enters **DEGRADED** mode.
+
+{brandname} provides different strategies for handling network partitions.
+The strategy that you choose depends entirely on application demands for data.
+
+Removing a network partition and merging the cluster back together means you need to choose between consistency or availability while {brandname} merges the partitions and resolves conflicts.
+If higher availability is your primary concern, {brandname} provides a merge strategy that allows read and write operations to continue.
+If consistency is more important than availability, you can deny read and write operations.
+There is also an option to allow reads only, which preserves consistency by denying writes to entries and provides availability as applications can still perform reads.
+
+Obviously while the {brandname} cluster is in **DEGRADED** mode, performance is not optimal and there is a cost associated with returning the cluster to a healthy state.
```

Comment:
Also different parts of the cluster might able to read/write entries and others not based on the configured policy